java/core/src/test/java/sleeper/core/partition/PartitionFactoryTest.java

```diff
@@ -128,9 +128,9 @@ void shouldSpecifyParentThenChildPartitionsWithTwoDimensions() {
         assertThat(nestedChildren).containsExactly(
                 Partition.builder()
                         .rowKeyTypes(rowKeyTypes)
-                        .region(Region.from(List.of(
-                                rangeFactory.createRange(key1, "aaa", null),
-                                rangeFactory.createRange(key2, "", "bbb"))))
+                        .region(new Region(List.of(
+                        rangeFactory.createRange(key1, "aaa", null),
+                        rangeFactory.createRange(key2, "", "bbb"))))
```

Comment:
The indentation here looks a bit strange. I guess checkstyle isnt picking it up since this is a test class?

Reply:
Fixed, thanks